pkg/util/log/clog.go

```diff
@@ -115,28 +115,26 @@ func SeverityByName(s string) (Severity, bool) {
 // terminals. Some terminals support 8 colors, some 256, others
 // none at all.
 type colorProfile struct {
-	infoPrefix  []byte
-	warnPrefix  []byte
-	errorPrefix []byte
-	timePrefix  []byte
+	infoPrefix  []Attribute
+	warnPrefix  []Attribute
+	errorPrefix []Attribute
+	timePrefix  []Attribute
 }
 
-var colorReset = []byte("\033[0m")
-
 // For terms with 8-color support.
 var colorProfile8 = &colorProfile{
-	infoPrefix:  []byte("\033[0;36;49m"),
-	warnPrefix:  []byte("\033[0;33;49m"),
-	errorPrefix: []byte("\033[0;31;49m"),
-	timePrefix:  []byte("\033[2;37;49m"),
+	infoPrefix:  []Attribute{0, 49, 36},
```

Comment:
Please keep the `[]byte` type in the struct definition above

Then call `format` here to pre-compute the byte arrays from the color codes.

(Since they are constant, it would be wasteful to re-compute them on every log message printed!!)